mhs-reference-implementation/mhs/routing/tests/test_sds.py

```diff
@@ -0,0 +1,96 @@
+from unittest import TestCase
+import mhs.routing.sds as sds
+import mhs.routing.tests.ldap_mocks as mocks
+from utilities.test_utilities import async_test
+import mhs.routing.routing_exception as re
+
+NHS_SERVICES_BASE = "ou=services, o=nhs"
+
+MHS_OBJECT_CLASS = "nhsMhs"
+
+PARTY_KEY = "AP4RTY-K33Y"
+SERVICE_ID = "urn:nhs:names:services:psis:MCCI_IN010000UK13"
+ODS_CODE = "ODSCODE1"
+
+
+expected_mhs_attributes = {
+    'nhsEPInteractionType': 'HL7',
+    'nhsIDCode': 'ODSCODE1',
+    'nhsMHSAckRequested': 'always',
+    'nhsMHSActor': ['urn:oasis:names:tc:ebxml-msg:actor:toPartyMSH'],
+    'nhsMHSDuplicateElimination': 'always',
+    'nhsMHSEndPoint': ['https://vpn-client-1411.opentest.hscic.gov.uk/'],
+    'nhsMHSIsAuthenticated': 'transient',
+    'nhsMHSPartyKey': 'AP4RTY-K33Y',
+    'nhsMHSPersistDuration': 'PT5M',
+    'nhsMHSRetries': '2',
+    'nhsMHSRetryInterval': 'PT1M',
+    'nhsMHSSyncReplyMode': 'MSHSignalsOnly',
+    'nhsMHsIN': 'MCCI_IN010000UK13',
+    'nhsMHsSN': 'urn:nhs:names:services:psis',
+    # 'nhsMhsCPAId': 'S918999410559', # Exclude this value since there seems to be two responses causing errors on this
```

Comment:
What do you mean by this?

Reply:
The sds call should return exactly 1 value for this lookup, however my opentest instance seems to return 2, this had been copied to the test data and caused an issue due to this value being an unique identifier. I've deleted the second instance in the test data so I've readded these attributes